src/Protocol/Protocol_1_8.cpp

```diff
@@ -504,7 +513,7 @@ void cProtocol_1_8_0::SendEntityAnimation(const cEntity & a_Entity, const Entity
 		return;
 	}
 
-	if (const auto AnimationID = GetProtocolEntityAnimation(a_Animation); AnimationID != static_cast<unsigned char>(-1))
+	if (const auto AnimationID = GetProtocolEntityAnimation(a_Animation); AnimationID != unsigned char(-1))
```

Comment:
`unsigned char(-1)`

Really, C++, why would you allow such ugly syntax...

Reply:
mmm keyword spaces

it does look 🤔 but i saw the PR used `type(literal)` syntax elsewhere so I made it consistent. do you want to change?

Reply:
No change necessary :)

Reply:
I WAS WRONG
c++ doesn't allow this notation to have spaces
i'll change it back